lib/templates.ts

```diff
@@ -1454,3 +1476,40 @@ export const checkIfClosed = async (formId: string) => {
     return null;
   }
 };
+
+export const updateNotificationsSetting = async (
+  formId: string,
+  notificationsInterval: NotificationsInterval
+) => {
+  const { user } = await authorization.canEditForm(formId).catch((e) => {
+    logEvent(
+      e.user.id,
+      { type: "Form", id: formId },
+      "AccessDenied",
+      "Attempted to update notifications interval for Form"
+    );
+    throw e;
+  });
+
+  if (!validateNotificationsInterval(notificationsInterval)) {
+    throw new Error(`Invalid notifications interval: ${notificationsInterval}`);
+  }
+
+  await prisma.template
+    .update({
+      where: {
+        id: formId,
+      },
+      data: {
+        notificationsInterval,
+      },
+    })
+    .catch((e) => prismaErrors(e, null));
+
+  logEvent(
+    user.id,
+    { type: "Form" },
```

Comment:
Please add the form ID to the audit log Subject.
```suggestion
    { type: "Form", id: formId },
```